pkg/vsphere/disk/disk.go

```diff
@@ -45,6 +45,40 @@ type Filesystem interface {
 	Unmount(path string) error
 }
 
+type ReferenceCount struct {
+	owner      string
+	refname    string
+	references int
+}
+
+func NewReferenceCount(o, n string) *ReferenceCount {
+	return &ReferenceCount{
+		owner:      o,
+		refname:    n,
+		references: 0,
+	}
+}
+
+func (r *ReferenceCount) Increment() {
+	r.references++
+	log.Infof("Increased %s references for %s to %d", r.refname, r.owner, r.references)
```

Comment:
This has to be a debug. However, I would even argue you need a log here at all. If you need to log a number of counts, I would add a String() methods for logging purposes somewhere else and I would log it where it is really necessary.